python/tank/platform/qt/tankqdialog.py

```diff
@@ -312,7 +312,12 @@ def _format_context_property(p, show_type=False):
             self.ui.btn_file_system.clicked.connect( self._on_filesystem )
             self.ui.btn_shotgun.clicked.connect( self._on_shotgun )
             self.ui.btn_reload.clicked.connect( self._on_reload )
-            
+
+            # When there is no file system locations, hide "Jump to File System" and "Jump to Shotgun" buttons.
+            if not self._bundle.context.filesystem_locations:
+                self.ui.btn_file_system.setVisible(False)
+                self.ui.btn_shotgun.setVisible(False)
```

Comment:
The _Jump to Shotgun_ button is not visible because the following is written in the ticket description:

> In the case of core, remove both jump to shotgun and jump to fs buttons altogether for simplicity: